CONTRIBUTING.md

```diff
@@ -63,119 +75,131 @@ This will open your documentation in a new browser tab at a local address (typic
 7. **Stop the Live Server**: When you're done testing or making changes, you can stop the Live Server by clicking the "Exit" button in the VS Code status bar.
 
 ## Adding a New Section to the Documentation
```

Comment:
This section is to add a new section to the intro course, and not guestbook.

Reply:
True @adiati98. I just need to figure out a new name to make it fit the guestbook. Any suggestions?

Reply:
It's not only the title, @CBID2.
Looking at the content in this section, this also for intro.
Guestbook doesn't require adding new section.

Reply:
Do you think it should be deleted then @adiati98?

Reply:
@CBID2, from reading the issue, that's what should happen in this PR. We should only be eliminating the two sections there. So that section becomes obsolete.